src/test/java/org/reaktivity/specification/nukleus/http2/streams/rfc7540/ConnectionManagementIT.java

```diff
@@ -38,196 +39,212 @@
     public final TestRule chain = outerRule(k3po).around(timeout);
 
     @Test
+    @ScriptProperty("serverTransport \"nukleus://http2/streams/source\"")
     @Specification({
             "${streams}/http.get.exchange/client",
             "${streams}/http.get.exchange/server"
     })
     public void httpGetExchange() throws Exception
     {
         k3po.start();
-        k3po.notifyBarrier("ROUTED_SERVER");
+        k3po.notifyBarrier("ROUTED_CLIENT");
```

Comment:
can you see how http is handling since it has both client and server impl ?

Reply:
I took the model from there. I don't see any `ROUTED_SERVER` in the tests under `org/reaktivity/specification/nukleus/http2/streams/rfc7540`.
I can see it only in tests under `org/reaktivity/specification/http/rfc7230`.

Can you please explain in more details what is there in http and is not here ?

Reply:
If the script is ambiguous on whether it is used for client or server, maybe we should make the barrier ambigous as well.  Go to Routed @jitsni ?

Reply:
No, we have two separate barriers. One is for client route and the other is for server route. Clients used in client mode should have ROUTED_CLIENT and clients used in server mode should have ROUTED_SERVER. What I am doing here is just correcting the barrier that needs to be used.

Maybe we can simplify by using one barrier, but this needs to be done on the entire spec. If you really want to do it, please add another task.

Reply:
If the answer is no that is ok with me.

To clarify the issue: what the [change in this PR](https://github.com/reaktivity/nukleus-http2.spec/pull/34/files/6b2edec3e3f6f33e3c2be1a7fd522ca00dcd4ff1#diff-2bb051368ab3a5161a4a786bbbc8d89dL19) introduces is removing the barrier from the accepted script. 

The client part of the script pair: ${streams}/http.get.exchange/client", "${streams}/http.get.exchange/server" is used in the implementation with client route script (CLIENT_ROUTED), and the server part is used with (SERVER_ROUTED).  IF we still want the barrier in the server script, then it seems like the best option is to rename the barrier to "ROUTED".  Another option is to have both barriers but then two notifies in this specification test.  

IF we are alright not having the barrier in the server script then I'm fine with the approach in this PR, note the change in this PR matches tls and http nukleus specs. BUT not http cache proxy tests, which uses ROUTED_PROXY as a generic name for the script pairs.

Reply:
The accept part of a k3po script is executed before running the k3po.start command (binding in case of TCP). I'm not even sure if a barrier on a accept is doing anything. Maybe just delaying the start of the conversation, accepting connections. Anyway, in our case, in our tests, that await barrier on the server.rpt is useless. The script runs the same with or without it.
@dpwspoon You told me several times to use http nukleus as a model, this is what I did. I cannot go through all nuklei to see how something is implemented, I'm having a hard time understanding one nukleus...

Reply:
@dpwspoon You wrote on slack to me:
> Though I’m okay with how you have done it, the missing understanding to me (and maybe Jitu) was that you removed SERVER_ROUTED from the other script pair, so the only script that had a barrier would only be used when testing the client implementation.

There is still a barrier also for server implementation. It is in the client.rpt used in that test. Basically, barriers are required only on client scripts, for each script according to its usage, in the client mode or server mode. All client scripts under rfc are used in server mode so they are waiting for ROUTED_SERVER and all client scripts under nukleus are used in client mode so they are waiting for ROUTED_CLIENT.

Reply:
Again, I'm ok with the approach taken here, just showing how two different ones did it :-)